lib/internal/streams/legacy.js

```diff
@@ -111,4 +119,40 @@ function prependListener(emitter, event, fn) {
     emitter._events[event] = [fn, emitter._events[event]];
 }
 
+let defaultHighwaterMark = null;
+ObjectDefineProperty(Stream, 'defaultHighwaterMark', {
```

Comment:
```suggestion
ObjectDefineProperty(Stream, 'defaultHighWaterMark', {
```
for naming consistency with existing APIs